CHANGELOG.md

```diff
@@ -2,7 +2,69 @@
 
 ## [Unreleased](https://github.com/CosmWasm/cw-plus/tree/HEAD)
 
-[Full Changelog](https://github.com/CosmWasm/cw-plus/compare/v0.10.3...HEAD)
+[Full Changelog](https://github.com/CosmWasm/cw-plus/compare/v0.11.0...HEAD)
+
+## [v0.11.0](https://github.com/CosmWasm/cw-plus/tree/v0.11.0) (2021-11-22)
+
+[Full Changelog](https://github.com/CosmWasm/cw-plus/compare/untagged-c9db5b9c4f390bdfe5f3...HEAD)
+
+**Breaking changes:**
+
+- Remove the primary key from the `MultiIndex` key specification [\#533](https://github.com/CosmWasm/cw-plus/issues/533)
+- `UniqueIndex` / `MultiIndex` key consistency [\#532](https://github.com/CosmWasm/cw-plus/issues/532)
+- Incorrect I32Key Index Ordering [\#489](https://github.com/CosmWasm/cw-plus/issues/489)
+- Deprecate `range` to `range_raw` [\#460](https://github.com/CosmWasm/cw-plus/issues/460)
+- How to completely clear a Map? [\#447](https://github.com/CosmWasm/cw-plus/issues/447)
```

Comment:
This don't belong here I think. And #551 is missing.

Reply:
It's under `Closed issues`, I didn't add `breaking` label in that issue.